article/app/services/MessageUsService.scala

```diff
@@ -36,8 +36,8 @@ class MessageUsService(messageUsS3Client: S3Client[MessageUsConfigData]) extends
       }
   }
 
-  def getBlogMessageUsConfigData(blogId: String): Option[MessageUsConfigData] = {
-    messageUsConfigData.get().flatMap(_.get(blogId))
+  def getBlogMessageUsConfigData(blogId: String): Option[MessageUsData] = {
```

Comment:
I wondered if this method should be renamed to align better to the return type? Something like `getBlogMessageUsData` maybe?